### ISSUE TITLE
training order

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/ppComplete#34]
Almost all these ideas are into release 4.0. stay super focused on 2.0 and 3.0 first. 

1. nn and RF first, as these generalize well
train the best nn until the rf finishes

only once these two have finished, start the next ones
don't try to start training them all in parallel. just two classifiers at a time. 

create the ability for people to add in more over time. so they can start with just a nn and a rf, and then train more later that night. 

consider modularizing this out into different repos entirely. have one repo for nns, one repo for RFs, etc. the master parent repo will then just aggregate all the child repos together intelligently. 

modularize even further: have one python formatting repo, which all other python repos include. 

i really want to make grid search for neural nets in JS available super broadly. 

offer three training speeds: thorough, medium, fast, and super light. just make it a 5 point scale. let people choose how much compute power they want to give to this. 

make the saved .pkl and .json files super modular- if they get one from a colleague, they should be able to swap that out in the directory, invoke rePredict and reEnsemble, and be good to go. 

if possible, it certainly never hurts to keep training the nn for longer :)

**Comment by ClimbsRocks** [comment]
no longer relevant now that we've moved away from brain.js
